apps/fhir/bluebutton/views/home.py

```diff
@@ -29,12 +30,14 @@ def fhir_conformance(request, via_oauth=False, *args, **kwargs):
     """
     crosswalk = None
     resource_router = get_resourcerouter()
-    call_to = resource_router.fhir_url
+    parsed_url = urlparse(resource_router.fhir_url)
 
-    if call_to.endswith('/'):
-        call_to += 'metadata'
+    call_to = None
+    if parsed_url.path is not None:
+        call_to = '{}://{}/{}/fhir/metadata'.format(parsed_url.scheme, parsed_url.netloc, kwargs.get('ver', 'v1'))
     else:
-        call_to += '/metadata'
+        # url with no path
+        call_to = '{}/{}/fhir/metadata'.format(resource_router.fhir_url, kwargs.get('ver', 'v1'))
```

Comment:
I think this might be where the missing v2 path is happening when rendering metadata?  Other endpoints use this code:
```
return "{}/{}/fhir/{}/".format(resource_router.fhir_url, 'v2' if self.version == 2 else 'v1',
                                           resource_type)
```

Currently the oauth-uri extension metadata JSON looks like this:
```
  "extension": [
          {
            "url": "http://fhir-registry.smarthealthit.org/StructureDefinition/oauth-uris",
            "extension": [
              {
                "url": "token",
                "valueUri": "http://localhost:8000/v1/o/token/"
              },
              {
                "url": "authorize",
                "valueUri": "http://localhost:8000/v1/o/authorize/"
              }
            ]
          }
        ]
```

Reply:
Will debug this one to see if it is on our side.

Reply:
made a fix and will push up soon